my.css

```diff
@@ -352,3 +352,153 @@ body {
     justify-content: center;
   }
 }
+
+/* Вторая секция */
+
+.bagscluth {
+  margin: 0;
+  padding: 0;
+  position: relative;
+  max-height: 900px;
+  display: flex;
+  background-color: #fae8e4;
+}
+.bagscluth__img {
+  width: 60%;
+  height: 844px;
+  margin: 0 auto;
+  display: flex;
+  align-items: center;
+}
+.bags-img {
+  position: absolute;
+  display: block;
+  width: 350px;
+  height: 380px;
+  padding-left: 100px;
+
+}
+.bags-img1 {
+  position: absolute;
+  width: 550px;
+  height: 750px;
+  left: 190px;
+}
+
+.inner {
+  width: 400px;
+  height: 500px;
+  position: relative;
+  padding: 30px;
+}
+.card {
+  position: absolute;
+  width: 400px;
+  height: 456px;
+  margin: 0 auto;
+  top: 200px;
+  display: flex;
+  flex-flow: column;
+  background-color: #fff;
+  border: 1px solid #fff;
+  box-shadow: 29px 30px 22px 0px rgba(0,0,0,0.07);
+}
+.card__header {
+  margin: 0 auto;
+}
+.subheading {
+  line-height: 1px;
+}
+.card__text {
+  padding: 25px;
+  margin: 0 auto;
+}
+.text {
+  opacity: 0.5;
+}
+.card__row {
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+}
+.card-clath-color {
+  width: 180px;
+  height: 40px;
+}
+.card-clath-color__button {
+  width: 30px;
+  height: 30px;
+  border-radius: 100%;
+}
+
+/* цвета */
+
+.card-clath-color__button_color_gray {
+  background-color: #dedede;
+}
+.card-clath-color__button_color_red{
+  background-color: #f29f9f;
+}
+.card-clath-color__button_color_blue{
+  background-color: #a8bff9;
+}
+.card-clath-color__button_color_green{
+  background-color: #90bd8f;
+}
+
+/* Карточка цен */
+
+.choice__row {
+  position: relative;
+  width: 380px;
+  height: 45px;
+}
+.card__choice-product {
+  position: absolute;
+  text-align: center;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 80px;
+  height: 40px;
+  left: 25px;
+  border: 1.5px solid #000000;
+  box-sizing: border-box;
+}
+.choice-product__buton {
+  background-color: #fff;
+  border: #fff;
+  padding: 10px;
+}
+
+.card-basket{
+ position: absolute;
+ left: 115px;
+ width: 255px;
+ height: 40px;
+ display: flex;
+ align-items: center;
+ justify-content: center;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/itog/pullRequest?prid=5642160)